composer.json

```diff
@@ -13,7 +13,7 @@
         "classmap": [ "app/AppKernel.php" ]
     },
     "require": {
-        "php": "^5.6.4||^7.0",
+        "php": "^7.0",
```

Comment:
Not sure if you are being sarcastic here or not...

Reply:
I'm meaning it. :blush: 
we will discuss that tomorrow with @soebbing

Reply:
@mitelg I see a few problems coming up there. Don't get me wrong, I'd love to have 7.1 as a minimum requirement. But I think, it should match the current shopware requirement. That however cannot easily be upgraded, because (up to version 5.4.x) you need ionCube support. Once that will be dropped, it will be much easier to increase the minimum version requirements.

Reply:
Ioncube supports php 7.1.x and 7.2.x:
https://www.ioncube.com/news.php

Reply:
ionCube does, but does it work also in combination with Shopware?

@mitelg can you confirm the documentation is outdated and we can switch to PHP 7.1 then?

Thanks

Reply:
@uehler @hranicka unfortunately the documentation is correct.
as @jkrzefski already mentioned, we can only encode up to PHP7.0 and we will not fix that, as we drop the ionCube encoding completely with Shopware 5.5.
So I row back and say PHP7.0 is totally fine, and maybe we will increase the requirement with an upcoming Shopware version.